channels/rdpsnd/client/winmm/rdpsnd_winmm.c

```diff
@@ -375,6 +386,8 @@ UINT freerdp_rdpsnd_client_subsystem_entry(PFREERDP_RDPSND_DEVICE_ENTRY_POINTS p
 		return CHANNEL_RC_NO_MEMORY;
 	}
 
+	winmm->volume = 0xFFFFFFFF;
+
```

Comment:
@hardening 

This is the application volume, not master volume. In fact OS default of the application volume is 100% so this should not be a problem.